packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -514,21 +531,19 @@ function persistLocalEnvConfig(context: $TSContext) {
       const awsSecretsFilePath = path.join(sharedConfigDirPath, awsSecretsFileName);
       JSONUtilities.writeJson(awsSecretsFilePath, awsSecrets);
 
-      awsInfo.awsConfigFilePath = awsSecretsFilePath;
+      (awsInfo as CredentialsEnvAwsInfo).awsConfigFilePath = awsSecretsFilePath;
     }
   }
 
-  const dotConfigDirPath = pathManager.getDotConfigDirPath();
-  const configInfoFilePath = path.join(dotConfigDirPath, constants.LocalAWSInfoFileName);
   const { envName } = context.exeInfo.localEnvInfo;
 
-  let envAwsInfo = {};
-  if (fs.existsSync(configInfoFilePath)) {
-    envAwsInfo = JSONUtilities.readJson(configInfoFilePath);
-  }
+  const envAwsInfo = stateManager.getLocalAWSInfo(undefined, { throwIfNotExist: false, default: {} });
```

Comment:
This change sets the AmplifyAppId and Region in the local-aws-info file for new environments